backend/core/src/main/java/org/sonarsource/sonarlint/core/NodeJsHelper.java

```diff
@@ -163,7 +161,7 @@ private String runSimpleCommand(Command command) {
     if (!stdErr.isEmpty()) {
       msg.append("\nstderr: ").append(String.join("\n", stdErr));
     }
-    logOutput.log(msg.toString(), LogOutput.Level.DEBUG);
+    LOG.debug("{}", msg);
```

Comment:
-> LOG.debug(msg);

Reply:
Nope, msg is a StringBuilder. `LOG.debug(msg.toString())` raises an issue about performance.
With slf4j2 we will be able to do:
`LOG.debug(() -> msg.toString())` to not pay the cost of the toString when DEBUG is not enabled